src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/CombinationController.php

```diff
@@ -249,16 +250,35 @@ private function formatListForPresentation(CombinationListForEditing $combinatio
                 'isSelected' => false,
                 'name' => $combination->getCombinationName(),
                 'reference' => $combination->getReference(),
-                //@todo: don't forget image path when implemented in the query
                 'impactOnPrice' => (string) $combination->getImpactOnPrice(),
                 'quantity' => $combination->getQuantity(),
                 'isDefault' => $combination->isDefault(),
+                'imageUrl' => $this->formatImageUrl($combination->getImagePath()),
             ];
         }
 
         return $data;
     }
 
+    /**
+     * @param string|null $imgPath
+     *
+     * @return string
+     */
+    private function formatImageUrl(?string $imgPath): string
+    {
+        $contextLink = $this->getContext()->link;
+
+        if (!$imgPath) {
+            $imageRetriever = new ImageRetriever($contextLink);
+            $defaultImg = $imageRetriever->getNoPictureImage($this->getContext()->language);
+
+            return $defaultImg['small']['url'];
```

Comment:
also this doesnt look safe, it will crash if 'small'|'url'  keys are not set. Not sure what to fallback to though. (I could simply provide empty string or a null, then src would have no picture, but wont crash at least)

Reply:
From the look of `ImageRetriever` the small one seems to be the safest one 😅
Although you can always plan for a safe guard:
```suggestion
            return $defaultImg['small']['url'] ?? '';
```